src/core/Footer/Footer.test.tsx

```diff
@@ -35,6 +35,20 @@ const urls: FooterElement[] = [
   },
 ];
 
+const customUrls: FooterElement[] = [
+  {
+    title: 'Custom link',
+    url: 'https://www.bentley.com/',
+  },
+  {
+    title: 'Products link',
+  },
+];
+
+const customUrlsFunction = () => {
+  return customUrls;
+};
```

Comment:
Is this helper needed? Could just inline it

Reply:
Fixed